index.bs

```diff
@@ -702,6 +723,14 @@ The <dfn attribute for=ScreenDetailed>onchange</dfn> attribute is the {{Screen/o
 
 When any [=screen/basic observable property=] or [=screen/advanced observable property=] of a [=/screen=] represented by a {{ScreenDetailed}} object |screenDetailed| changes, [=/queue a global task=] on the [=/relevant global object=] of |screenDetailed| using the [=/window placement task source=] to [=/fire an event=] named <dfn event for=ScreenDetailed>`change`</dfn> at |screenDetailed|.
 
+<!-- ====================================================================== -->
+### {{ScreenDetailed/onhdrheadroomchange}} attribute ### {#api-screendetailed-onhdrheadroomchange-attribute}
+<!-- ====================================================================== -->
+
+The <dfn attribute for=ScreenDetailed>onhdrheadroomchange</dfn> attribute is an [=/event handler IDL attribute=] whose [=/event handler event type=] is {{ScreenDetailed/hdrheadroomchange!!event}}.
+
+When the [=screen/HDR headroom=] of a [=/screen=] represented by a {{ScreenDetailed}} object |screenDetailed| changes, [=/queue a global task=] on the [=/relevant global object=] of |screenDetailed| using the [=/window placement task source=] to [=/fire an event=] named <dfn event for=ScreenDetailed>`hdrheadroomchange`</dfn> at |screenDetailed|.
```

Comment:
It might be worth a (non-normative?) explanation that hdrHeadroom changes will not fire ScreenDetailed change events, because changes may be high-frequency. Perhaps the spec should also explicitly describe that implementers can limit the granularity of values and/or frequency of changes. Perhaps its explicit omission should also be called out with the definition of advanced observable property too?

Reply:
Good point. Added:
- a note in the "observable screen properties" that HDR headroom is present in neither because it updates independently and at high frequency
- a note in onhdrheadroomchanged to the same effect.

I've updated the PR to include these changes